content/blog/pulumi-components/index.md

```diff
@@ -639,25 +639,31 @@ if __name__ == "__main__":
 1. Define a `main.go` file
 2. Declare an instance of `NewProviderBuilder`,  passing in a name, namespace and the components being built
 
+{{% notes type="info" %}}
+The code below uses the new pulumi-go-provider v1 APIs. Make sure you are using the latest version of `github.com/pulumi/pulumi-go-provider`.
+{{% /notes %}}
+
 ```go
 package main
 
 import (
+    "context"
+
     "github.com/pulumi/pulumi-go-provider/infer"
 )
 
 func main() {
-    err := infer.NewProviderBuilder().
-            WithName("go-components").
+    prov, err := infer.NewProviderBuilder().
             WithNamespace("your-org-name").
             WithComponents(
-                infer.Component(MyComponent),
+                infer.ComponentF(MyComponent),
```

Comment:
What does the "F" suffix signify?

Reply:
It is a convention in this go library that the argument is a function. Usually to distinguish from an ordinary value.

For example we have RunProvider and RunProviderF, variants of how you start a provider. One takes a provider instance, the other takes a "factory" func that produces a provider instance.